react/components/InfoCard/index.js

```diff
@@ -118,40 +120,42 @@ const InfoCard = ({
   )
 
   return (
-    <div className={containerClasses} style={containerStyle}>
-      <div className={textContainerClasses}>
-        {headline && (
-          <h1
-            className={`${
-              styles.infoCardHeadline
-            } t-heading-2 mt6 ${alignToken}`}
-          >
-            {headline}
-          </h1>
+    <LinkWrapper imageActionUrl={imageActionUrl} extraCondition={!isFullModeStyle} linkProps={{ className: 'no-underline' }}>
+      <div className={containerClasses} style={containerStyle}>
+        <div className={textContainerClasses}>
+          {headline && (
+            <h1
+              className={`${
```

Comment:
Using classNames and separate in `headlineClasses` variable would be better, I guess.